parm/parm_gdas/aero_crtm_coeff.yaml

```diff
@@ -1,23 +1,23 @@
 mkdir:
 - !ENV ${DATA}/crtm/
 copy:
-- - !ENV ${FV3JEDI_FIX}/crtm/${CRTM_VER}/AerosolCoeff.bin
+- - !ENV ${FV3JEDI_FIX}/crtm/${crtm_VERSION}/AerosolCoeff.bin
   - !ENV ${DATA}/crtm/AerosolCoeff.bin
-- - !ENV ${FV3JEDI_FIX}/crtm/${CRTM_VER}/CloudCoeff.bin
+- - !ENV ${FV3JEDI_FIX}/crtm/${crtm_VERSION}/CloudCoeff.bin
   - !ENV ${DATA}/crtm/CloudCoeff.bin
-- - !ENV ${FV3JEDI_FIX}/crtm/${CRTM_VER}/v.viirs-m_npp.SpcCoeff.bin
+- - !ENV ${FV3JEDI_FIX}/crtm/${crtm_VERSION}/v.viirs-m_npp.SpcCoeff.bin
   - !ENV ${DATA}/crtm/v.viirs-m_npp.SpcCoeff.bin
-- - !ENV ${FV3JEDI_FIX}/crtm/${CRTM_VER}/v.viirs-m_npp.TauCoeff.bin
+- - !ENV ${FV3JEDI_FIX}/crtm/${crtm_VERSION}/v.viirs-m_npp.TauCoeff.bin
   - !ENV ${DATA}/crtm/v.viirs-m_npp.TauCoeff.bin
-- - !ENV ${FV3JEDI_FIX}/crtm/${CRTM_VER}/v.viirs-m_j1.SpcCoeff.bin
+- - !ENV ${FV3JEDI_FIX}/crtm/${crtm_VERSION}/v.viirs-m_j1.SpcCoeff.bin
   - !ENV ${DATA}/crtm/v.viirs-m_j1.SpcCoeff.bin
-- - !ENV ${FV3JEDI_FIX}/crtm/${CRTM_VER}/v.viirs-m_j1.TauCoeff.bin
+- - !ENV ${FV3JEDI_FIX}/crtm/${crtm_VERSION}/v.viirs-m_j1.TauCoeff.bin
   - !ENV ${DATA}/crtm/v.viirs-m_j1.TauCoeff.bin
-- - !ENV ${FV3JEDI_FIX}/crtm/${CRTM_VER}/NPOESS.VISice.EmisCoeff.bin
+- - !ENV ${FV3JEDI_FIX}/crtm/${crtm_VERSION}/NPOESS.VISice.EmisCoeff.bin
   - !ENV ${DATA}/crtm/NPOESS.VISice.EmisCoeff.bin
-- - !ENV ${FV3JEDI_FIX}/crtm/${CRTM_VER}/NPOESS.VISland.EmisCoeff.bin
+- - !ENV ${FV3JEDI_FIX}/crtm/${crtm_VERSION}/NPOESS.VISland.EmisCoeff.bin
   - !ENV ${DATA}/crtm/NPOESS.VISland.EmisCoeff.bin
-- - !ENV ${FV3JEDI_FIX}/crtm/${CRTM_VER}/NPOESS.VISsnow.EmisCoeff.bin
+- - !ENV ${FV3JEDI_FIX}/crtm/${crtm_VERSION}/NPOESS.VISsnow.EmisCoeff.bin
   - !ENV ${DATA}/crtm/NPOESS.VISsnow.EmisCoeff.bin
-- - !ENV ${FV3JEDI_FIX}/crtm/${CRTM_VER}/NPOESS.VISwater.EmisCoeff.bin
+- - !ENV ${FV3JEDI_FIX}/crtm/${crtm_VERSION}/NPOESS.VISwater.EmisCoeff.bin
```

Comment:
We are moving away from using `ENV` here and use `$()` instead.
I can make these fixes if that is desired.

Reply:
No problem.
I can make a suggestion on this PR.